homework.10/homework.10/circleViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  circleViewController.swift
+//  homework.10
+//
+//  Created by Stanislau on 03.04.2022.
+//
+
+import UIKit
+
+class circleViewController: UIViewController {
```

Comment:
We should name classes, structs, enums and so on starting with big letter: `CircleViewController`